include/robot_localization/common_utilities.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ROBOT_LOCALIZATION_COMMON_UTILITIES_H
+#define ROBOT_LOCALIZATION_COMMON_UTILITIES_H
+
+#include<fstream>
```

Comment:
Just to keep the style consistent, can we add a space in between the include directive and the file name?